Dockerfile

```diff
@@ -0,0 +1,116 @@
+FROM registry.suse.com/suse/sle15:15.6 AS fetcher
```

Comment:
@paleloser Why not `registry.suse.com/bci/bci-base:15.6`?

Reply:
There's not a reason for use one or another, but they both look the same:

[registry.suse.com/suse/sle15:15.6](https://registry.suse.com/repositories/suse-sle15sp6): `sha256:497869098014f9e869e4a2d36f3074d1505e200185ee8d005410a6bebbffb9d9`
[registry.suse.com/bci/bci-base:15.6](https://registry.suse.com/repositories/bci-bci-base-15sp6): `sha256:497869098014f9e869e4a2d36f3074d1505e200185ee8d005410a6bebbffb9d9`

Reply:
We can switch to the other name if you prefer it for some reason

Reply:
IMHO, `bci/bci-base` is clearer.

Reply:
Done dc86dab